ticktick/oauth2.py

```diff
@@ -244,7 +269,8 @@ def _post(self, url, **kwargs):
             RunTimeError: If the request could not be completed.
         """
 
-        response = self.session.post(url, **kwargs)
+        response = self.session.request("POST",url, **kwargs)
+        print(response.content)
```

Comment:
I think you left an accidental print statement in the commit here.

Reply:
Thx